tests/x11/yast2_snapper.pm

```diff
@@ -0,0 +1,92 @@
+use base "x11test";
+use testapi;
+
+# Test for basic yast2-snapper functionality. It assumes the data of the
+# opensuse distri to be available at /home/$username/data (as granted by
+# console_setup.pm)
+
+# Helper for letting y2-snapper to create a snapper snapshot
+sub y2snapper_create_snapshot() {
+    my $self = shift;
+    my $name = shift || "Awesome Snapshot";
+   # Open the 'C'reate dialog and wait until it is there
+    send_key "alt-c";
+    assert_screen 'yast2_snapper-createsnapshotdialog', 100;
+    # Fill the form and finish by pressing the 'O'k-button
+    type_string $name;
+    sleep 2;
+    send_key "tab";
+    sleep 2;
+    send_key "tab";
+    sleep 2;
+    send_key "tab";
+    sleep 2;
+    type_string "a=1,b=2";
+    sleep 2;
+    send_key "alt-o";
+}
+sub run() {
+    my $self = shift;
+    # Make sure yast2-snapper is installed (if not: install it)
+    ensure_installed "yast2-snapper";
+    # Start an xterm as root
+    x11_start_program("xterm");
+    wait_idle;
+    become_root;
+    # Create a snapper config
+    script_run "btrfs subvolume create /tmp/testdata";
+    script_run "snapper create-config /tmp/testdata";
```

Comment:
Since snapper is deactivated on small harddrives by default it is actually not enabled because of our current openQA settings. so I have to create a snapper config anyway.
I decided to go to an extra subvolume to be sure that I only see the changes I do. Just using /tmp leads to some extra changes because yast creates some files there.